problems/1.types+operators.js

```diff
@@ -2,15 +2,20 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+const isNumber = (val) => {
+  let(val < 23)
+return "is val a number"
 }
 
+
+
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
+function isTruthy(val) {
+  let(val + 0 === 0)
+  return "Is val truthy"
```

Comment:
This function will always return"Is val truthy". (besides the error from the let). YOu're being given an argument (which you're calling val) YOu have to check for truthyness. Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false. 
You can use a condtional if to help you. if (val) return true else return false